python/BioSimSpace/Sandpit/Exscientia/Process/_amber.py

```diff
@@ -1093,9 +1229,15 @@ def getElectrostaticEnergy(self, time_series=False, block="AUTO"):
         energy : :class:`Energy <BioSimSpace.Types.Energy>`
            The electrostatic energy.
         """
-        return self.getRecord("EELECT", time_series, _Units.Energy.kcal_per_mol, block)
+        return self.getRecord(
```

Comment:
Might add a doc to explain the different keys.

Reply:
Good idea. Originally this was only an internal function so the user didn't really need to know what the inputs meant. Once I have the mapping in place, then I'll just explain that they are AMBER output keys and point the user to the manual. (I'm not going explain them all, since they are not even documented appropriately in the manual itself.)